sycl/test-e2e/OptionalKernelFeatures/fp64-conv-emu.cpp

```diff
@@ -1,9 +1,8 @@
-// REQUIRES: ocloc, gpu, linux
+// REQUIRES: ocloc, gpu, linux, arch-intel_gpu_dg2_g10
 // UNSUPPORTED: cuda, hip
 
-// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen  -Xsycl-target-backend "-device pvc" -fsycl-fp64-conv-emu -O0 %s -o %t_opt.out
-// TODO: Enable when GPU driver is updated.
-// RUNx: %{run} %t_opt.out
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_dg2_g10 -fsycl-fp64-conv-emu -O0 %s -o %t_opt.out
```

Comment:
Why the test execution was moved from PVC to DG2 (G10)? Has it executed before (and currently in main branch) on PVC?

Reply:
We don't have PVC HW in our intel/llvm CI. Note that AOT-compiled device images for Intel GPUs can be run on any Intel GPU, simply because SPIR-V is always embedded into them, so they fallback to JIT.

Reply:
Do we still need to execute this test on PVC internally, or this is not required in such case?

Reply:
> Do we still need to execute this test on PVC internally, or this is not required in such case?

I'm not sure if selecting PVC was a right thing to do in the first place - it supports `fp64` and therefore would test nothing in this specific case.

Reply:
In that case, sounds good to me to switch to DG2, thanks for explanation